BREAKING.md

```diff
@@ -37,6 +38,16 @@ Developers using the component will need to migrate to a virtual scroll solution
 
 Any references to the virtual scroll types from `@ionic/core` have been removed. Please remove or replace these types: `Cell`, `VirtualNode`, `CellType`, `NodeChange`, `HeaderFn`, `ItemHeightFn`, `FooterHeightFn`, `ItemRenderFn` and `DomRenderFn`.
 
+<h4 id="version-7x-slides">Slides</h4>
+
+`ion-slides`, `ion-slide`, and the `IonicSwiper` module have been removed from Ionic.
```

Comment:
Is `IonicSwiper` a module? It appears to be a constant of shape:
```
{
  name: string;
  on: {
    afterInit(swiper: any): void;
  }
```

Reply:
IonicSwiper was a Swiper plugin before they changed how the plugins work in Swiper 6 or 7. IonicSlides is the new one.

Reply:
Should this be updated instead to:

> `ion-slides`, `ion-slide` and the `IonicSwiper` plugin have been removed from Ionic.

With module I immediately think of an Angular module being removed here.